File: lab3-semester1/1.cpp
// C++ Program to convert Temprature from degree Centigrade into Fahrenheit
#include <iostream>

using namespace std;

int main()
{
    float cen, fahr;
    cout<<"Input temprature in Centigrade: ";
    cin>>cen;//takes input
    
     //formula to convert Centigrade to Fahrenheit
    fahr = (1.8 * cen) + 32.0;
    
    //formula to convert Fahrenheit to Centigrade
    //cen = (fahr - 32) / 1.8; 
    
    cout<<"Result: "<<fahr<<" degree in Fahrenheit"<<endl;
    
    system("pause");
    return 0;
}

File: lab3-semester1/1.go
//Program to convert Temprature from degree Centigrade into Fahrenheit

package main

import "fmt"

func main() {
    var cen, fahr float64
    fmt.Print("Input Temprature in Centigrade: ")
    fmt.Scanln(&cen)
    
    fahr=(1.8*cen)+32.0
    //formula to convert from Cen. to Fahr.
    
    //also you can use formula to convert from Fahrenheit to Centigrade
    //cen = (fahr - 32) / 1.8
    
    fmt.Println("Result: ", fahr, "degree in Fahrenheit")
 }
